arrow-libs/optics/arrow-optics/build.gradle.kts

```diff
@@ -20,6 +20,10 @@ if (enableCompatibilityMetadataVariant) {
   }
 }
 
+tasks.withType<org.jetbrains.kotlin.gradle.tasks.KotlinCompile>().configureEach {
+  kotlinOptions.freeCompilerArgs += listOf("-Xenable-builder-inference")
+}
+
```

Comment:
AFAIK this is not needed. Since we have BuilderInference already working with *EffectScope in core and dont have to add it

Reply:
Just pushed a commit removing it